Makefile

```diff
@@ -10,13 +10,28 @@ INSTALL_DATA = $(INSTALL) -m 644
 
 prefix = /usr/local
 
+# ifdef check allows the user to pass custom dirs
+# as per the README
+
 # The directory to install todo.sh in.
-bindir = $(prefix)/bin
+ifdef INSTALL_DIR
+	bindir = $(INSTALL_DIR)
+else
+	bindir = $(prefix)/bin
+endif
 
 # The directory to install the config file in.
-sysconfdir = $(prefix)/etc
-
-datarootdir = $(prefix)/share
+ifdef CONFIG_DIR
+	sysconfdir = $(CONFIG_DIR)
+else
+	sysconfdir = $(prefix)/etc
+endif
+
+ifdef BASH_COMPLETION
```

Comment:
the readme has both BASH_COMPLETION and BASH_COMPLETION_DIR. should probably be fixed there as well